app/src/main/java/uk/ac/ebi/atlas/controllers/rest/DASFeaturesController.java

```diff
@@ -71,6 +70,10 @@ public String getDifferentialJsonResults(
 
         model.addAttribute("geneId", geneId);
         model.addAttribute("geneName", geneName);
+        var baseUrl = "http://www.ebi.ac.uk/";
```

Comment:
I think it is bad practice to keep the url here. What would we do for wwwdev or www-test?